torch/csrc/jit/mobile/interpreter.cpp

```diff
@@ -28,13 +34,35 @@ bool InterpreterState::run(Stack& stack) {
     //      }
     //    }
     Instruction inst = code_->instructions_[pc];
-    TORCH_CHECK(isOpSupportedInMobile(inst.op), toString(inst.op),
-                " is not supported in mobile module.");
     switch (inst.op) {
       case OP: {
         c10::Dispatcher::singleton().callBoxed(*code_->operators_[inst.X], &stack);
         ++pc;
       } break;
+      case OPN: {
```

Comment:
This isn't the right place to do this resolution. There should be a code_->vararg_operators_ table that holds `void fn(int N, Stack stack)` functions that this instruction invokes.